### DISCUSSION TITLE
Should I rename it?

### DISCUSSION THREAD

**Discussion by jdx** [jdx/mise#1306] [Polls]
rtx was a working title I never changed. I get a lot of understandable complaints about rtx due to conflicts with nvidia. The conflicts make it hard to search both on google and places like slack and twitter.

Personally, I don't really care if it means anything as long as it's easy to type and pronounce (no consecutive duplicate characters, ideally 3 but possibly 4 characters).

 If you have any suggestions on a new name, lmk. We're also chatting about potential names in our discord. I'm a little unsure if I want to go through with a name change but I do think if this poll ends up with a majority voting for it I think it's likely to happen. It'll be some effort, and it would involve some work from you users to change—that said, if ultimately we decide to change the name obviously the sooner the better.

**Comment by booniepepper** [comment]
The creator of Make didn't change from tabs to spaces because he had 12 users.

_(Or something like that. The story is related in at least "The Unix Haters Handbook" and "Masterminds of Programming" with some details changing between versions)_

Anyway if you wanna change it, it's cheaper to change now than when it's got a million users

**Comment by booniepepper** [comment]
P.S.
> It's a terrible name for obvious reasons.

IDK I always thought `rtx` is a fine name.

**Reply by jdx** [reply]
the problem is the conflict with nvidia, other than that there isn't anything wrong with it

**Reply by jdx** [reply]
updated the description with a bit more clarity around that

**Comment by JonathanPlasse** [comment]
J'adore le nouveau nom
